kubernetes/README.md

```diff
@@ -67,37 +67,41 @@ Then, you will need to provide the Google Service Account key file, also generat
 kubectl create secret generic workspace-credentials --from-file=/path/to/<keyfile>.json
 ```
 
-## Configuring the SCIM bridge
-
-You'll need to edit the `op-scim-config.yaml` file and change the variable `OP_LETSENCRYPT_DOMAIN` to the domain you've decided on for your SCIM bridge. This allows LetsEncrypt to issue your deployment an SSL certificate necessary for encrypted traffic.
-
 ## Deploy to the Kubernetes cluster
 
-Run the following `kubectl` command to complete your deployment. It will deploy the both the `redis` server and the `op-scim` app.
+Run the following command deploy SCIM bridge:
 
 ```bash
-cd scim-examples/kubernetes/
 kubectl apply -f .
 ```
 
-## Configuring the DNS entries
+## Create the DNS record
 
-The Kubernetes deployment creates a public load balancer in your environment pointing to the SCIM bridge.
+The [`op-scim-bridge` Service](./op-scim-service.yaml) creates a public load balancer attached to your cluster that forwards TLS traffic to SCIM bridge.
 
-To get its public IP address:
+Run the following command, and copy the address listed under the `External IP` column for to the `op-scim-bridge` Service from the output:
 
 ```bash
-kubectl describe service/op-scim-bridge 
-# look for ‘LoadBalancer Ingress’
+kubectl get svc
 ```
 
-It can take some time before the public address becomes available.
+Note: It can take a few minutes before the public address becomes available. Run the command again if doesn't appear in the output.
 
-At this stage, you can finish configuring your DNS entry as outlined in [PREPARATION.md](/PREPARATION.md).
+Create a public DNS record pointing to this address as outlined in [the preparation guide](/PREPARATION.md).
+
+## Configure Let's Encrypt
+
+After the DNS record above has propagated, run the following command to set the `OP_LETSENCRYPT_DOMAIN` environment variable to the fully-qualified domain name (FQDN) for SCIM bridge based on this record (replace `scim.example.com` with the FQDN):
+
+```bash
+kubectl set env deploy/op-scim-bridge OP_LETSENCRYPT_DOMAIN=scim.example.com
```

Comment:
Question: given we are no longer setting this value statically into the config, does this persist past redeployments? 

I'll validate this in my testing, but in case you know immediately offhand.

Reply:
It does indeed! 👍